### ISSUE TITLE
Profile Listing Template Profile Image Alt Text Editoria11y Warning

### ISSUE THREAD

**Issue by TaylorYeeBC** [BellevueCollege/bc-sitka-spruce-department-theme#445]
The Editoria11y plugin highlights that the alt text of Profile images in Profile Listing Template pages are redundant. We found in the twig file that the Alt Text is being automatically created with "Picture of `Person's Name`" and probably should be adjusted.

Line of code found with Taija: [https://github.com/BellevueCollege/bc-sitka-spruce-department-theme/blob/trunk/views/content/subcomponents/profile-list-element.twig#L21](https://github.com/BellevueCollege/bc-sitka-spruce-department-theme/blob/trunk/views/content/subcomponents/profile-list-element.twig#L21)

Staging site page used with Profile Listing Template: [https://staging-bc2022redesigncontentstage-stage.kinsta.cloud/taylor-sitka/2nd-profile-listing/](https://staging-bc2022redesigncontentstage-stage.kinsta.cloud/taylor-sitka/2nd-profile-listing/)

Image in Media Library Alt Text

<img width="800" height="428" alt="Image" src="https://github.com/user-attachments/assets/2e6e1cae-0cad-4593-8e0f-ff3378adafd6" />

Alt Text for same Image used in a Profile

<img width="2559" height="1374" alt="Image" src="https://github.com/user-attachments/assets/65d2743b-d881-4b31-9484-f1c565dfc579" />

Alt Text on Profile Listing Page w/ Editoria11y Warning

<img width="2558" height="1372" alt="Image" src="https://github.com/user-attachments/assets/d24e8b7b-3e37-437c-a88b-0bbb39542046" />
